docs/quickstart.md

```diff
@@ -250,6 +250,7 @@ At this point, we’ve created a principal and granted it the ability to manage
 
 To use a Polaris-managed catalog in [Apache Spark](https://spark.apache.org/), we can configure Spark to use the Iceberg catalog REST API.
 
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
why is the next line disabled?

Reply:
Link checker gets HTTP 403 (unauthorized) error when trying to access https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-spark - maybe mvnrepository.com doesn't allow for web traffic originating from GitHub Actions?

here's link to the failure in my repo: https://github.com/bbrewington/fork-polaris/actions/runs/10342235075/job/28624929825#step:4:62

Reply:
you're right, wget doesnt work either 
> wget https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-spark

Reply:
looks like it's getting blocked by a Cloudflare policy

when I run:

```bash
curl https://mvnrepository.com/artifact/org.apache.iceberg/iceberg-spark
```

I see some elements in the response starting with `cf_chl` (i.e. [Cloudflare Challenge](https://stackoverflow.com/questions/61415857/dealing-with-cloudflares-cf-chl-jschl-tk-cf-chl-captcha-tk))

Looks like this one definitely needs to be an exception, and I think I'll revert the "don't check this line" and instead just include this link as an exception in `.github/workflows/check-md-link-config.json`